net/netfilter/nf_tables_api.c

```diff
@@ -8636,6 +8637,16 @@ static int nf_tables_abort(struct net *net, struct sk_buff *skb,
 	ret = __nf_tables_abort(net, action);
 	nft_gc_seq_end(nft_net, gc_seq);
 
+	WARN_ON_ONCE(!list_empty(&net->nft.commit_list));
+
+	/* module autoload needs to happen after GC sequence update because it
+	 * temporarily releases and grabs mutex again.
+	 */
+	if (action == NFNL_ABORT_AUTOLOAD)
+		nf_tables_module_autoload(net);
+	else
+		nf_tables_module_autoload_cleanup(net);
+
```

Comment:
This code is actually a part of this later commit in this series:
https://github.com/ctrliq/kernel-src-tree/pull/8/commits/c2e9b8c60919d326fe966de61676f361df723297
Lets move this section to that area since at this commit we're doing this action in two places now.

This is the upstream for that commit: https://github.com/torvalds/linux/commit/0d459e2ffb541841714839e8228b845458ed3b27

When we move this we can update the commit message for this commit to not include the `upstream-diff`

Reply:
Good catch! Resolved in the next push.